### DISCUSSION TITLE
Cant find my domain after transefering the domain.

### DISCUSSION THREAD

**Discussion by sondrealf** [vercel/community#6332] [Help]
### Summary

I transferred the domain from dreamhost to here. It says in who.is that it has been transferred. But i cannot find the domain in my tab. I have allready contacted Dreamhost support, they says that it is in your control. 

I know i wrote right "Auth Code". The domain is "alfnes.dev"

### Steps to Reproduce

Transfer a domain from dreamhost to vercel. You will first get an mail that Dreamhost needs to verify the transation, afterwards who.is will update but it will not update in the profile page.

**Comment by amyegan** [comment]
Do you see the domain in your account settings at [vercel.com/account/domains](https://vercel.com/account/domains)?
